### PR TITLE
feat: Adding address field for AccountInfo

### LINKED ISSUE #700: Adding address field for AccountInfo
## Describe the problem you'd like to have solved

In protocol buffers `blockchain.proto` we have `AccountInfo` message for response rpc method `GetAccount and GetAccountByNumber` but this message don't have address field similar `ValidatorInfo`

### Describe the ideal solution

Idea solution for this problem, we need add address field for response `GetAccount and GetAccountByNumber`.

### Alternatives and current work-around

In `ValidatorInfo` currently response address.

### PR DESCRIPTION
**Pull request by kehiy**
## Related issue(s)

If this Pull Request is related to an issue, mention it here.
- Fixes #700

### CODE REVIEW

**Comment by b00f** [comment]
@kehiy 

Better we remove [accountByNumber](https://github.com/pactus-project/pactus/blob/1627d50b2b6d13b2e62e131da3974970f73bc115/store/account.go#L66) It is useless now

**Comment by kehiy** [comment]
@b00f it's done.

File: store/account.go [hunk 1]
```diff
@@ -63,15 +63,6 @@ func (as *accountStore) account(addr crypto.Address) (*account.Account, error) {
 	return nil, ErrNotFound
 }
 
-func (as *accountStore) accountByNumber(number int32) (*account.Account, error) {
-	acc, ok := as.numberMap[number]
```
Comment (b00f) [inline store/account.go:67]: Do we need to keep `numberMap`?

Comment (kehiy) [reply store/account.go:67]: nope, It's removed.

**Review by b00f — APPROVED** [review]

**Pull request merged**